test/rules/curly/never/test.ts.lint

```diff
@@ -0,0 +1,24 @@
+if (so) {
```

Comment:
There's one edge case that needs to be handled. You can't remove the curly braces in the following example

```ts
if (foo) {
    if (bar)
        foo(bar);
} else
    bar();
```